runtime/compiler/il/J9Node.hpp

```diff
@@ -284,6 +284,11 @@ class OMR_EXTENSIBLE Node : public OMR::NodeConnector
    bool chkDontInlineUnsafePutOrderedCall();
    const char * printIsDontInlineUnsafePutOrderedCall();
 
+   /**
+    * Checks  and return true if the callNode is JNI Call to Unsafe.copyMemory
+    */
+   bool isUnsafeCopyMemoryIntrinsic();
```

Comment:
This API seems very very specific for it to exist in the `Node` class. I worry this class will turn into a kitchen sink eventually. Does this really need to exist? It seems it gets used in two locations:

1. In `processJNICall` which we can just replace with:

```
   if (comp->canTransformUnsafeCopyToArrayCopy() &&
       (methodSymbol->getRecognizedMethod() == TR::sun_misc_Unsafe_copyMemory ||
        methodSymbol->getRecognizedMethod() == TR::jdk_internal_misc_Unsafe_copyMemory0))
```

2. In `transformUnsafeCopyMemoryCall` which we can just inline a partial version of this API.

Are we sure introducing this extra complexity into the `Node` class outweighs the benefits?

Reply:
@fjeremic  Reason I chose to introduce this API in Node class is that it contains all the information that it needed to verify if the JNI call can be transformed. Idea behind using this intrinsic is that if in later version of JDK if there are changes in the call chain, we only need to make changes in one place. 
There is another potential use of this API in recognized call transformer to identify and transform this call as you suggested in https://github.com/eclipse/openj9/pull/10366#discussion_r471784886 , which I think we should pursue and remove the complexity and duplicate code from the codegen.

Reply:
That sounds reasonable. Thanks for explaining further uses.